.github/BOTMETA.yml

```diff
@@ -706,7 +706,7 @@ files:
     labels:
       - clustering
       - k8s
-  $module_utils/keycloak.py:
+  $module_utils/identity/keycloak/:
     maintainers: eikef
```

Comment:
Can you add `support: community` to this?

Reply:
Done in 17204fce9a156c291b3f832cec75a82f4903048e.